packages/recomponents/src/components/r-badge/r-badge.scss

```diff
@@ -54,6 +54,10 @@
         overflow: hidden;
         max-width: 100%;
         text-overflow: ellipsis;
+        line-height: var(--badge-tag-line-height);
+        margin: var(--badge-tag-margin);
+        padding: var(--badge-tag-padding);
+        box-shadow: none;
```

Comment:
This is the style for r-badge, just note that the r-badge is used in Recomm everywhere.

Reply:
It is for badges with type "tag", that are used in `r-select`